java/bogdana_maksiuta/src/com/zoolatech/lecture2/tasks/_1/Calculator.java

```diff
@@ -0,0 +1,70 @@
+package com.zoolatech.lecture2.tasks._1;
+
+import java.util.Scanner;
+
+public class Calculator {
+    Scanner scanner = new Scanner(System.in);
+
+    public void processOfCalculation() {
```

Comment:
Methods processOfCalculation, inputNumber, inputSymbol should be a part of the calculator API (see task's description). They can be defined in the Main class, and later you can use the calculator instance to perform a specific operation.

Reply:
fixed